src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -296,15 +296,15 @@ public static function get_user_data_from_custom_post_type( $theme, $should_crea
 	 *
 	 * @since 5.9.0
 	 *
-	 * @return WP_Theme_JSON Entity that holds user data.
+	 * @return WP_Theme_JSON Entity that holds styles for the current theme.
 	 */
-	public static function get_user_data() {
+	public static function get_global_styles() {
```

Comment:
```suggestion
	public static function get_user_data() {
```

Reply:
Note that there's already a `get_user_data` so this needs to be different, hence, why I suggested `get_user_data_{something}`

Reply:
oh, dismiss my comment. Too many changes, I got confused :)

Reply:
No worries @oandregal. Yes, it reverted it back to the original 👍